docs/HyperIndex/overview.md

```diff
@@ -49,6 +49,20 @@ While HyperIndex gives you a complete indexing solution with schema management a
 
 ---
 
+## Hypersync API Token Requirements
+
+Starting from **9 May 2024**, HyperSync (the data engine powering HyperIndex) will implement rate limits for requests without API tokens. Here's what you need to know:
```

Comment:
We don't mention at all what the rate limits might be. We might want to mention, its rate limited to appox 100 req per minute and this means potentially your indexer may be 2x slower without a token. To avoid simply add a token?

Reply:
We haven't decided on this yet.

Reply:
Yeah, I thought leave this vague. But maybe be more direct in saying that if you want guaranteed uninterupted support use a token.

It is also hard to be precise. It is local rate limitting per hypersync instance, so for ETH we have 4 replicas now (we will reduce this in the future) - but that means 4x the rate limit. So best we can say is between 100 and 200 requests per minute or something like that.

For an indexer it'll only affect the height polling, don't think it'll make things slower for historical sync (at least at the levels of rate limiting we will have to start)